.clang-tidy

```diff
@@ -0,0 +1,26 @@
+---
+# NOTE: there must be no spaces before the '-', so put the comma last.
+InheritParentConfig: true
+Checks: '
+abseil-*,
+bugprone-*,
+-bugprone-easily-swappable-parameters,
+clang-analyzer-*,
+cppcoreguidelines-*,
```

Comment:
maybe also consider [clang-analyzer-optin.performance.Padding](https://clang.llvm.org/extra/clang-tidy/checks/clang-analyzer/optin.performance.Padding.html) 

Otherwise, this looks pretty good.

Reply:
> maybe also consider [clang-analyzer-optin.performance.Padding](https://clang.llvm.org/extra/clang-tidy/checks/clang-analyzer/optin.performance.Padding.html)

It's enabled. `clang-analyzer-*` enables all `clang-analyzer` related checks.

Reply:
Oh missed that.